src/Core/src/Platform/iOS/MauiUIApplicationDelegate.Menu.cs

```diff
@@ -29,13 +29,15 @@ public override void BuildMenu(IUIMenuBuilder builder)
 			MenuBuilder = builder;
 
 			UIWindow? window = null;
-			if (OperatingSystem.IsMacCatalystVersionAtLeast(14))
+
+			// WindowingBehaviors is only available on iOS 16+ and Mac Catalyst
+			if (OperatingSystem.IsMacCatalystVersionAtLeast(16) || OperatingSystem.IsIOSVersionAtLeast(16))
```

Comment:
I think this changes the logic? We need to keep the iOS 14 here, but wrap the inner if block on lines 51/53 with this check. We still need to have this block for the fallback on line 66 which is in this main block.